proto/xudrpc.proto

```diff
@@ -75,7 +75,7 @@ service Xud {
     };
   }
   
-  /* Shutdown the xud daemon. */
+  /* Begin shutting down the xud daemon. */
```

Comment:
Isn't XUD = Exchange Union Daemon? What about removing the redundant `daemon` here?

Reply:
Yeah maybe i'll rename "daemon" to "server"? Or just drop daemon and say "Begin shutting down xud"?

Reply:
I would prefer just `Begin shutting down xud` but it is up to you

Reply:
Done, I also changed a comment in the code from "Inititate graceful shutdown of the server" to "Initiate graceful shutdown of xud"